Backpack/src/main/java/net/skyscanner/backpack/checkbox/BpkCheckbox.kt

```diff
@@ -78,6 +78,8 @@ open class BpkCheckbox @JvmOverloads constructor(
         intArrayOf(textDisabledColor, textEnabledColor)
       )
     )
+
+    setPaddingRelative(resources.getDimensionPixelSize(R.dimen.bpkSpacingSm), paddingTop, paddingEnd, paddingBottom)
```

Comment:
the view padding is different to the compose padding because the view one already includes some default padding within the drawable, but it ends up being the same